library.json

```diff
@@ -18,6 +20,8 @@
     },
 
     "platforms": "*",
+    
+    "frameworks": "arduino",
```

Comment:
As per https://docs.platformio.org/en/latest/librarymanager/config.html#frameworks I would prefer to use '*' as I don't assume this is arduino specific library.